lib/room.js

```diff
@@ -636,6 +643,8 @@ function handleSignalingEvents(room, signaling) {
 
   signaling.on('dominantSpeakerChanged', () => room.emit('dominantSpeakerChanged', room.dominantSpeaker));
 
+  signaling.on('liveTranscription', data => room.emit('liveTranscription', data));
```

Comment:
Thanks 😃  I missed adding this in my initial PR.

Reply:
Happy to help!